java-core/solid/fruit-shop-tests.md

```diff
@@ -56,3 +56,6 @@ public void afterEachTest() {
     Storage.storage.clear();
 }
 ```  
+#### Try to avoid NPE in your code
+We need to make our code workable and we shouldn't get RuntimeException in runtime.
```

Comment:
I'd better say not about Runtime in general, but about common Runtime exceptions. 'You could expect your custom runtime exception, but not one of common ones, such as NPE, IndexOutOfBounds etc.'